services/users/src/routes/team.ts

```diff
@@ -0,0 +1,102 @@
+import express from "express";
+import { asyncHandler } from "@api/common/src/middleware";
+import { BadRequestError } from "@api/common/src/errors";
+
+import { TeamModel, Team } from "../models/team";
+
+export const teamRoutes = express.Router();
+
+teamRoutes.route("/").post(
+  asyncHandler(async (req, res) => {
+    const { name, event, members, description, publicTeam } = req.body;
```

Comment:
When we create a new team, the only default member should be the user making the request. So you. can remove members from the post body.